### ISSUE TITLE
Cli auth failure

### ISSUE THREAD

**Issue by csnuknet** [expo/expo-cli#664]
Every attempt made to build:ios currently results in the following auth error with the following build:

`  Expo CLI 2.18.7 environment info:
    System:
      OS: macOS 10.14.5
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 12.1.0 - /usr/local/Cellar/node/12.1.0/bin/node
      Yarn: 1.15.2 - /usr/local/bin/yarn
      npm: 6.9.0 - /usr/local/bin/npm
    IDEs:
      Xcode: 10.2.1/10E1001 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^32.0.6 => 32.0.6 
      react: ^16.8.1 => 16.8.1 
      react-native: https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz => 0.57.1 
      react-navigation: ^3.0.9 => 3.2.1 
    npmGlobalPackages:
      expo-cli: 2.18.7`

```
Authentication with Apple Developer Portal failed!
Reason: Unknown reason, raw: "Unauthorized Access"
Error: Reason: Unknown reason, raw: "Unauthorized Access"
    at _callee$ (/expo-cli@2.18.7/src/commands/build/ios/appleApi/fastlane.js:24:17)
    at tryCatch (/usr/local/lib/node_modules/expo-cli/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/usr/local/lib/node_modules/expo-cli/node_modules/regenerator-runtime/runtime.js:296:22)
    at Generator.prototype.<computed> [as next] (/usr/local/lib/node_modules/expo-cli/node_modules/regenerator-runtime/runtime.js:114:21)
    at step (/usr/local/lib/node_modules/expo-cli/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
    at /usr/local/lib/node_modules/expo-cli/node_modules/babel-runtime/helpers/asyncToGenerator.js:28:13
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
```
There are no policy/privacy notices in any portal and all keys/permissions are set, seem to have hit a complete brick wall, ideas?

**Comment by cruzach** [comment]
Could you try logging into App Store Connect? There may be some new agreements you'll have to review, and that's what's causing this issue

**Comment by csnuknet** [comment]
> Could you try logging into App Store Connect? There may be some new agreements you'll have to review, and that's what's causing this issue

We've completed the free app agreement and are not making use of the paid app agreement (none pending/new)

**Comment by brentvatne** [comment]
did you ever get past this issue? i'm not seeing other users encountering it, i wonder if it was an intermittent issue or if it ended up being some account configuration issue

**Comment by BenjaminWFox** [comment]
I'm having a similar issue, although the stack track is not exactly the same. Seems to be the same with any of the iOS commands.

This is on a newly created organization account. Things I've tried:
- Logged in to itunesconnect, which brought up a flow asking if I wanted to sell music, books, or videos (but not apps) which I couldn't complete because I'm not trying to sell any of those things 🤷‍♂️
- Logged in to appstoreconnect, created a new bundle, then created a new app (filled out seller name, etc...)
- Poked around for any other agreements that looked unsigned, couldn't find any.

From the `credentials:manager` > `Add new Distribution Certificate` or `build:ios` commands - 
```
Requesting session...
Authentication with Apple Developer Portal failed!
Error: Reason: Unknown reason, raw: "Unauthorized Access"
    at runAction (/expo-cli@3.13.1/src/appleApi/fastlane.ts:28:17)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at authenticate (/expo-cli@3.13.1/src/appleApi/authenticate.ts:46:40)
    at Context.ensureAppleCtx (/expo-cli@3.13.1/src/credentials/context.ts:56:24)
    at generateDistCert (/expo-cli@3.13.1/src/credentials/views/IosDistCert.ts:496:3)
    at CreateIosDist.provideOrGenerate (/expo-cli@3.13.1/src/credentials/views/IosDistCert.ts:68:12)
    at CreateIosDist.create (/expo-cli@3.13.1/src/credentials/views/IosDistCert.ts:40:25)
    at CreateIosDist.open (/expo-cli@3.13.1/src/credentials/views/IosDistCert.ts:45:22)
    at CredentialsManager.run (/expo-cli@3.13.1/src/credentials/route.ts:54:12)
    at runCredentialsManagerStandalone (/expo-cli@3.13.1/src/credentials/route.ts:8:3) {
  rawDump: 'Unauthorized Access'
}
```

When I have time I'll also try contacting apple developer support and see if they can find anything unfinished with the account.

**Comment by BenjaminWFox** [comment]
Lol, I should have held off on that last post another 5 minutes. I had one more thing to try that I didn't really want to since it's kind of a pain but...here you go 🤦‍♂️

The developer account (apple id) I was using was _not associated with any device_. I was only getting the MFA notifications on my phone.

On my macbook I signed out of my main apple id and logged in with the id I was trying to use with Expo. Immediately tried `build:ios`, and after entering credentials I got the prompt on the mac (not on my phone), which has the **extra step** of clicking `allow` before getting the MFA code.

Now everything works fine.
